example/settings.json

```diff
@@ -22,8 +22,8 @@
   "cowToken": {
     "salt": "0x0000000000000000000000000000000000000000000000000000000000001337"
   },
-  "virtualCowToken": {
-    "gnoPrice": "375000000000000",
-    "nativeTokenPrice": "37500000000000"
-  }
-}
+  "gnoPrice": "375000000000000",
+  "nativeTokenPriceOnETH": "37500000000000",
+  "nativeTokenPriceOnGnosisChain": "37500000000000",
+  "multiTokenMediatorGnosisChain": "0x8888888888888888888888888888888888888888"
```

Comment:
Probably, it is nicer to hardcode these values, similar to USDC tokens etc. I will update the PR